lib/runtime/src/system_status_server.rs

```diff
@@ -254,7 +255,10 @@ async fn metrics_handler(state: Arc<SystemStatusState>) -> impl IntoResponse {
     // Update the uptime gauge with current value
     state.update_uptime_gauge();
 
-    // Get metrics from the registry
+    // Empty hierarchy means this is the top-level metrics at the DistributedRuntime level
+    state.drt().execute_metrics_callbacks("");
```

Comment:
question: why the need for the hierarchy support if we are just doing top level - is it recursive?